person/views.py

```diff
@@ -287,7 +287,7 @@ def godfather_update(request,godfather_slug):
                 messages.error(request, str(v.args[0]))
         else:
             messages.error(request, 'Formulario con errores')
-    return render(request, 'godfather_form.html', {"form": form})
+    return render(request, 'person/godfather/godfather_form.html', {"form": form})
```

Comment:
I think we are naming the form files as "form.html" only, maybe I am wrong but I have been doing like this 😳

Reply:
Okay, I will rename it. I just changed the path because it was on `/templates`, instead of `/templates/person/godfather`